features/steps/given.php

```diff
@@ -169,15 +175,9 @@ function($world) {
 	}
 );
 
-$steps->Given( '/^a local wp-cli Composer repository$/',
-	function ( $world ) {
-		$world->composer_add_wp_cli_local_repository();
-	}
-);
-
-$steps->Given( '/^a WordPress install with Composer$/',
+$steps->Given( '/^a dependency on current wp-cli$/',
```

Comment:
Just for reference:
The `current wp-cli` can later be extracted as a variable, so that one can have alternatives like `a dependency on stable wp-cli` or `a dependency on wp-cli v1.0.0`.
I think the PR is good for now and we should only do this if we indeed need it for additional tests.